spec/syncSpec.js

```diff
@@ -120,6 +120,30 @@ describe('Sync', function() {
     });
   });
 
+  it('should cause a collision', function() {
+    // The UID of the record which should have a collision.
+    var recordId;
+
+    return manage()
+    .then(doCreate)
+    .then(waitForSyncEvent('remote_update_applied'))
+    .then(function verifyUpdateApplied(event) {
```

Comment:
We need these to get the actual UID of the record from the sync server, currently we only possess the temporary hash UID which is replaced by a MongoDB update. 

This is needed to make the call to the `/dataset/:datasetId/record/:recordId` endpoint on the server to update the MongoDB record.